src/formatDate.js

```diff
@@ -8,7 +8,61 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const FROM_SEPARATOR_FORMAT = fromFormat[3];
+  const TO_SEPARATOR_FORMAT = toFormat[3];
+  let year, month, day;
```

Comment:
Let's create an object with keys that match format ('DD', 'MM').

Then in the second loop you will be able just take the correct value like this: `dateObject[part]`